src/workerd/jsg/util.c++

```diff
@@ -341,6 +351,16 @@ void throwInternalError(v8::Isolate* isolate, kj::Exception&& exception) {
   }
 }
 
+void addExceptionDetail(Lock& js, kj::Exception& exception, v8::Local<v8::Value> handle) {
+  js.tryCatch([&]() {
+    Serializer ser(js);
```

Comment:
I assume we don't need to worry about the Serializer wire format version.

Reply:
At present, this isn't actually sent over the wire regardless. But I should probably specify a version explicitly, as we do elsewhere, so that V8 updates don't create temporary backward-incompatibility.